### ISSUE TITLE
Adding ACES as working profile

### ISSUE THREAD

**Issue by Beep6581** [Beep6581/RawTherapee#2083]
Originally reported on Google Code with ID 2099

```
ACES is a colorspace that is widely used in the film industry. Since OpenEXR is planned
to be integrated in RT, using ACES as working profile (and output too if i've understood
correctly) might be interesting for people wanting to use RT in their pipeline, would
it be by Gimp2.9 (which support 32 bits images) or any other tools from this industry.

Could some color specialist evaluate the possibility of adding ACES to RT? Or is it
a too big change?

http://www.oscars.org/science-technology/council/projects/aces.html
http://ampas.github.io/
```

Reported by `natureh.510` on 2013-12-02 00:56:22

**Comment by Beep6581** [comment]
For what it's worth, I generated an ACES colorspace using ArgyllCMS or dispcalGUI:
http://filebin.net/vaz44pm8c2/aces.icc

**Comment by Beep6581** [comment]
"OpenColorIO (OCIO) is a complete color management solution geared towards motion
picture production with an emphasis on visual effects and computer animation.
OCIO provides a straightforward and consistent user experience across all
supporting applications while allowing for sophisticated back-end configuration
options suitable for high-end production usage. OCIO is compatible with the
Academy Color Encoding Specification (ACES) and is LUT-format agnostic,
supporting many popular formats."

https://github.com/imageworks/OpenColorIO

**Comment by iliasg** [comment]
.. Dcraw 9.27 supports exporting at ACES color space !!

**Comment by Beep6581** [comment]
ACES and DCI-P3 added as output profiles in issue #3782

**Comment by Beep6581** [comment]
ACES AP0 and AP1 are now available as working spaces in 5.5.
